client/src/components/LoggedIn.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
Yep we can figure out what to do this this later, maybe just needs to move to a different component - maybe when they search for a restaurant